recipes-samples/images/lmp-feature-jobserv.inc

```diff
@@ -0,0 +1,11 @@
+# Packages required for jobserv compatibility
+CORE_IMAGE_BASE_INSTALL += " \
+    python3-compression \
+    python3-distutils \
+    python3-json \
+    python3-multiprocessing \
+    python3-netclient \
+    python3-pkgutil \
+    python3-shell \
+    python3-unixadmin \
```

Comment:
Only python3-json and python3-multiprocessing are required, was able to run jobserv without the others.

Reply:
done